node.go

```diff
@@ -305,6 +336,15 @@ func formatBytes(i int64) (result string) {
 	var n float64
 	sFmt, eFmt := "%.01f", ""
 	switch {
+	case i > (1024 * 1024 * 1024 * 1024 * 1024 * 1024):
```

Comment:
I think it will be more readable to use consts like these:

```go
const (
	_           = iota
	KB float64 = 1 << (10 * iota)
	MB
	GB
	TB
	PB
	EB
	ZB
	YB
)
```